tests/linkedlist_tests.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from dsalgo.linked_list import LinkedList
+
+
+class TestList(unittest.TestCase):
+    def setUp(self):
+        self.ls = LinkedList()
+        self.ls.append(1)
+        self.ls.append(2)
+        self.ls.append(5)
+
```

Comment:
for good testing practice, you can use random module and generate a list of random numbers to test